styles.css

```diff
@@ -14,6 +14,21 @@ input.in_box_button[type="submit"] {
     margin-left: 0;
 }
 
+.linedrawer {
```

Comment:
In all plugins you should always use more precise css identifiers. Identifiers like that might break the theme of the whole moodle installation.

Reply:
@rtschu Adding a mention-tag here, since it is really important!